api/src/permissions/modules/validate-access/validate-access.ts

```diff
@@ -17,9 +17,13 @@ export interface ValidateAccessOptions {
  * optional primary keys. This is done by reading the item from the database using the access
  * control rules and checking if we got the expected result back
  */
-export async function validateAccess(options: ValidateAccessOptions, context: Context) {
+export async function validateAccess(
+	options: ValidateAccessOptions,
+	context: Context,
+	skipCollectionExistsCheck = false,
```

Comment:
Can we add this as an optional option in the `ValidateAcessOptions`? Seems like that is the place where it should live IMO. Easier to understand than a positional parameter.

Reply:
Absolutely! 💯 Refactored in https://github.com/directus/directus/pull/23991/commits/c52206b04dd4d306162b354f7ed86b65ed2ffd36